community/capstone/APKBUILD

```diff
@@ -2,16 +2,17 @@
 # Maintainer: Valery Kartel <valery.kartel@gmail.com>
 pkgname=capstone
 pkgver=4.0.1
-pkgrel=0
+pkgrel=1
 pkgdesc="The Ultimate Disassembler"
 options="!check" # Testsuite fails
 url="https://www.capstone-engine.org"
 arch="all"
 license="BSD-3-Clause-Clear"
 depends_dev="$pkgname"
 makedepends="bash py3-setuptools cython3 python3-dev"
-subpackages="$pkgname-dev py3-${pkgname}:_py3 cstool"
-source="$pkgname-$pkgver.tar.gz::https://github.com/aquynh/capstone/archive/$pkgver.tar.gz"
+subpackages="$pkgname-dev $pkgname-static py3-${pkgname}:_py3 cstool"
```

Comment:
In the current version of abuild there's a bug requiring you to put -static subpackage in front of the -dev one.

Reply:
i noticed and fixed it.